protocol-designer/src/steplist/formLevel/handleFormChange/utils.ts

```diff
@@ -669,24 +679,28 @@ const getLiquidClassValuesMoveLiquid = (args: {
   const { path, tipRack, volume: rawVolume } = rawForm
   const volume = Number(rawVolume)
   const {
-    positionReference: aspiratePositionReference,
-    offset: aspirateOffset,
     flowRateByVolume: aspirateFlowRateByVolume,
     preWet,
     mix: aspirateMix,
     delay: aspirateDelay,
   } = aspirate
+  const {
+    positionReference: aspiratePositionReference,
+    offset: aspirateOffset,
+  } = aspirate.aspiratePosition
```

Comment:
nit, can you deconstruct aspiratePosition from aspirate like in line 682 and then this could be: 

```suggestion
  const {
    positionReference: aspiratePositionReference,
    offset: aspirateOffset,
  } = aspiratePosition
```

its just cleaner this way to read and we prefer this :D